models/mirror.go

```diff
@@ -403,14 +403,6 @@ func SyncMirrors() {
 				continue
 			}
 
-			// Create reference
-			if result.oldCommitID == GIT_SHORT_EMPTY_SHA {
-				if err = MirrorSyncCreateAction(m.Repo, result.refName); err != nil {
-					log.Error(2, "MirrorSyncCreateAction [repo_id: %d]: %v", m.RepoID, err)
-				}
-				continue
```

Comment:
I think all we need is just to remove this single line of "continue", everything else should works automatically.

Reply:
If the continue is removed then the code will fail later when trying to retrieve the full commitID for GIT_SHORT_EMPTY_SHA which does not exist.